bindings/wysiwyg-ffi/src/ffi_composer_model.rs

```diff
@@ -18,6 +18,13 @@ impl ComposerModel {
         }
     }
 
+    pub fn set_html(self: &Arc<Self>, html: String) -> Arc<ComposerUpdate> {
```

Comment:
I went for replace_all_html to make it clear the whole thing is replaced. I think we should try to be consistent - if you think set_html is better, maybe we should change everything to that?

Reply:
Yeah `replace_all_html` seems fine to me, I wonder if this should also reset undo/redo states (perhaps with an extra parameter), as I suppose there might be cases where we either want to push a state to history or we want to completely remove old states (right now it does nothing - which means that if we play with the editor, then replace_all_html, undo we will get to state at N-2).

Reply:
Hmm, good point. What do we want right now? We should implement the behaviour we want now (either delete history or add this to history properly), and add an option later.

Reply:
I suppose right now we want to simply clear history, as I don't expect to do something other than setting up the initial HTML at start.
Still not sure there is a real case for preserving undo. (perhaps something around switches between plain/WYSIWYG, if we want to save or keep the model alive in these kind of cases)

Reply:
Yeah, let's leave that use case until we need it, and clear the undo/redo history for now.